components/builder-depot/Cargo.toml

```diff
@@ -64,7 +64,7 @@ git = "https://github.com/habitat-sh/urlencoded.git"
 branch = "habitat"
 
 [dependencies.zmq]
-git = "https://github.com/erickt/rust-zmq.git"
+git = "https://github.com/habitat-sh/rust-zmq.git"
```

Comment:
currently the PR was the master branch of the submitter's repo so putting it here seemed safest but aree a comment will be good. Hopefully this repo won't live long.